signal-strength/signal-strength.go

```diff
@@ -0,0 +1,67 @@
+/*
+management-interface - Web based management of Raspberry Pis over WiFi
+Copyright (C) 2018, The Cacophony Project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package signalstrength
+
+import (
+	"encoding/xml"
+	"io/ioutil"
+	"net/http"
+	"net/http/cookiejar"
+)
+
+const (
+	modemAddress = "http://192.168.8.1"
+	statusPath   = "/api/monitoring/status"
```

Comment:
You could just make this: `statusURL = modemURL + "/api/monitoring/status"`

This is probably a little clearer and then there's no need to concatenate in the code.